ops/storage-ddls/cassandra-setup.cql

```diff
@@ -57,8 +57,6 @@ CREATE TABLE trustedsources (
    PRIMARY KEY (pipelinekey, externalsourceid, sourcetype, rank)
 );
 
-DROP TABLE IF EXISTS events;
-DROP TYPE IF EXISTS features;
```

Comment:
Good catch, moved everything together in f9c7fdc. Ran it against cqlsh and seems to work.